support-frontend/assets/components/directDebit/directDebitPopUpForm/directDebitPopUpForm.tsx

```diff
@@ -43,9 +39,6 @@ function DirectDebitPopUpForm(props: PropTypes): JSX.Element {
 		return (
 			<div className="component-direct-debit-pop-up-form">
 				<div className="component-direct-debit-pop-up-form__content">
-					<h1 className="component-direct-debit-pop-up-form__heading">
-						<PageTitle phase={props.phase} />
-					</h1>
```

Comment:
The form looks slightly odd without this heading, but there’s not much point replacing it when it won’t be there in the final design.